draft-ietf-idr-5g-edge-service-metadata-24.xml

```diff
@@ -510,6 +576,9 @@
         associated with the Side-ID, is generated by the egress routers with
         the egress routers' loopback address as the NLRI.</t>
 
+	<!-- XXX JMH - I've flagged this before: You don't go from having TLV
+	to not having TLV in a protocol.  Make this a TLV even if the length is
+	fixed. -->
```

Comment:
Need help here. Not sure how to address this comment

Reply:
Your PDU component here is:
sub-type 2 octets
reserved + i bit 2 octets
site-id 2 octets
site availability percentage 2 octets

It should be:
sub-type 2 octets
+++ length 1 octets
+++ reserved + i bit 1 octets
site-id 2 octets
site availability percentage 2 octets

In theory, you're asking for early allocation. This is a big change. It's the right thing to do before we move forward, but it means code MUST be updated.

Reply:
got it.  Added the Length field.

Reply:
` 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
|      PhyAvailIdx Sub-Type     | Length        |I|   Reserved  |
+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
|        Site-ID (2 octets)     | Site Availability Percentage  |
+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+`